hibernate-core/src/main/java/org/hibernate/dialect/CockroachDB1920Dialect.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.dialect;
+
+import org.hibernate.dialect.identity.CockroachDB1920IdentityColumnSupport;
+import org.hibernate.dialect.identity.IdentityColumnSupport;
+import org.hibernate.hql.spi.id.MultiTableBulkIdStrategy;
+import org.hibernate.hql.spi.id.inline.InlineIdsInClauseBulkIdStrategy;
+import org.hibernate.type.descriptor.sql.*;
+
+import java.sql.*;
+
+import java.sql.DatabaseMetaData;
+
+/**
+ * An SQL dialect for CockroachDB 19.2 and later. This is the first dialect for CockroachDB. It extends
```

Comment:
Should we change ' for CockroachDB 19.2 and later' to ' for CockroachDB 19.2'? From the classname it implies so.

Reply:
The dialect would still work with later versions of CockroachDB. Also, I used the same language that other dialects have (e.g. PostgreSQL82Dialect).